runtime/gc_base/ContinuationObjectList.cpp

```diff
@@ -46,13 +46,22 @@ MM_ContinuationObjectList::MM_ContinuationObjectList()
 }
 
 MM_ContinuationObjectList *
-MM_ContinuationObjectList::newInstanceArray(MM_EnvironmentBase *env, uintptr_t arrayElements)
+MM_ContinuationObjectList::newInstanceArray(MM_EnvironmentBase *env, uintptr_t arrayElementsTotal, MM_ContinuationObjectList *listsToCopy, uintptr_t arrayElementsToCopy)
 {
 	MM_ContinuationObjectList *continuationObjectLists;
```

Comment:
either set this to null or just join in with the next line
same for other 3 lists